debian/rules

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --no-parallel --verbose
+
+override_dh_auto_build:
+	if [ ! -f ./libraries.h5p ]; then \
+		echo "ERROR: Please create the libraries.h5p before building the debian package by running:"; \
+		echo "    ./pack_dependencies"; \
+		exit 1; \
+	fi
+	npm install
```

Comment:
I think here we should use `package-list.json`, so let's do `npm ci` instead of `npm install`

Reply:
** `package-lock.json`